extension/src/openvic-extension/classes/GFXButtonStateTexture.cpp

```diff
@@ -141,16 +137,11 @@ StringName const& GFXButtonStateTexture::button_state_to_name(ButtonState button
 	static const StringName name_selected = "selected";
 	static const StringName name_error = "INVALID BUTTON STATE";
 	switch (button_state) {
-		case HOVER:
-			return name_hover;
-		case PRESSED:
-			return name_pressed;
-		case DISABLED:
-			return name_disabled;
-		case SELECTED:
-			return name_selected;
-		default:
-			return name_error;
+	case HOVER:	   return name_hover;
+	case PRESSED:  return name_pressed;
+	case DISABLED: return name_disabled;
+	case SELECTED: return name_selected;
+	default:	   return name_error;
```

Comment:
Again, using tabs for complex alignment produces inconsistent behaviour, using spaces between the `:` and `return` statements would work better.